drivers/modem/Kconfig

```diff
@@ -139,6 +139,24 @@ config MODEM_CELL_INFO
 	help
```

Comment:
BTW, we do not do merge branches in zephyr history so please rebase the PR.

Reply:
Will do.